src/sync/installation.ts

```diff
@@ -245,6 +246,8 @@ const doProcessInstallation = async (data: BackfillMessagePayload, sentry: Hub,
 		throw new Error(`Error processing task: installationId=${installationId}, repositoryId=${nextTask.repositoryId}, task=${task}`);
 	};
 
+	const gitHubAppConfig = await getGitHubAppConfig(subscription, logger);
```

Comment:
I know the code is a bit tricky here. So it seems this `getNextTask` is called only in this file, and the caller position all contains the `BackfillMessagePayload`, which in turns contains the `gitHubAppConfig?` ( as they are msg in the queue). 

So maybe just grab it, and avoid one db call?